test/eslint-config.js

```diff
@@ -0,0 +1,165 @@
+module.exports = {
```

Comment:
Why it's in the `test` folder? Should it be a special `eslint` folder to be extracted to the packages repository later?

Reply:
It's a temp thing. I want to have it inside `WordPress/packages` :)

Reply:
I'm moving to `eslint` folder because Jest loads this file as it was a test 😃